components/ReusedStyles.js

```diff
@@ -1,45 +1,61 @@
 import styled from "styled-components";
 
-const PrimaryInfo = styled.p`
-  font-size: 1rem;
+const PrimaryInfoMap = styled.p`
+  font-family: "Inconsolata";
```

Comment:
You should always declare a fallback. 

From [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family): "You should always include at least one generic family name in a font-family list, since there's no guarantee that any given font is available. This lets the browser select an acceptable fallback font when necessary."

```suggestion
  font-family: "Inconsolata", monospace;
```